filebeat/beater/filebeat.go

```diff
@@ -126,9 +126,13 @@ func (fb *Filebeat) Run(b *beat.Beat) error {
 	<-fb.done
 
 	if fb.config.ShutdownTimeout > 0 {
+		logp.Info("Shutdown timeout %v", fb.config.ShutdownTimeout)
```

Comment:
when using `Shutdown timeout started` I would assume `%v` to be `time.Now()`, right?